common/decisions/wc_lordaeron_decisions.txt

```diff
@@ -0,0 +1,73 @@
+﻿
+develop_the_undercity = { 
+    ai_check_interval = 6
+	sort_order = 1
+	picture = "gfx/interface/illustrations/decisions/wc_undercity_throne_room_decision.dds"
+	desc = develop_the_undercity_desc
+	selection_tooltip = develop_the_undercity_tooltip
+	major = yes
+
+	is_shown = {
+		NOT = {
+			title:c_lordaeron = { has_variable = develop_the_undercity }
+		}
+		is_of_forsaken_like_culture = yes
+		has_title = title:c_lordaeron
+	}
+
+	is_valid = {
+		piety_level >= 2
+
+		prestige_level >= 2
+
+		religion = { is_in_family = rf_shadow }
+
+		is_of_forsaken_like_culture = yes
+	}
+
+	cost = { 
+		gold = 250
+		prestige = 500
+	}
+
+	effect = {
+		title:c_lordaeron = { set_variable = develop_the_undercity }
```

Comment:
```suggestion
		title:c_lordaeron = { set_variable = { name = develop_the_undercity years = 7 } }
```
I would make like this and completely remove all code in `on_title_lost` on action for performance. It will be checked anyway 2 years later.

Reply:
ok, will put back like it was

Reply:
Ah, actually no. Ignore this part, keep like it is now. I forgot we used it fore global flag replacement. I am tired so cant think normally